doc/tut3.md

```diff
@@ -253,6 +253,30 @@ The call to `myMacro` will generate the following code:
   echo MyType(a: 123.456'f64, b: "abcdef")
   ```
 
+You may also use `ident` in cases in which you wish to create an identifier from
```

Comment:
This whole section should be **after** "Building Your First Macro" and should get a section name of its own.